### ISSUE TITLE
Modified date of property does not change if only image updated.

### ISSUE THREAD

**Issue by mp3duck** [reapit/foundations#7646]
When making a call to "`/properties`" end-point, the "`modified`" field does not update to be the date/time that the image was updated.
For example, a given property is returned in a search specifying a "from" and "to" modified date, but one or more of the images has a modified date outside of this range. (a time greater than the main property modified date).

I'd expect the property modified date to change when any image or document related to it also changes.

**Comment by AshDeeming** [comment]
Hi @mp3duck 
Do you have an example of this please?

**Comment by mp3duck** [comment]
yes, but I don't want to share the calls on a public forum, as it's an example of live data.
Is there a way to share privately, please?

**Comment by AshDeeming** [comment]
Thanks @mp3duck 
Please send any sensitive data to the below, with the GitHub reference as the subject:
sinfo@reapitfoundations.zendesk.com
